.github/workflows/build-release.yml

```diff
@@ -30,6 +30,13 @@ jobs:
           npm install --prefix ./_dev
           npm run vite:build --prefix ./_dev
 
+      - name: Install composer dev dependencies for tests
+        working-directory: tests
+        run: composer install --ansi --prefer-dist --no-interaction --no-progress
+
+      - name: Apply header stamps
+        run: php tests/vendor/bin/header-stamp --license=tests/vendor/prestashop/header-stamp/assets/afl.txt --exclude=vendor,node_modules,storybook/var
```

Comment:
What's the point in applying the headers on all the files if we had a check before merging pull-requests?

If you meant to update built files before release, it may be worth a comment.